docs/FAQ.md

```diff
@@ -82,7 +82,7 @@ Remember this is a free/open community support. We may not be available 24/7 to
 
 - **Question 8**: CPU are not kept busy (like 10% CPU usage only) in Windows when using LightGBM on very large datasets with many core systems.
 
-- **Solution 8**: Please use [use Visual Studio](https://github.com/Microsoft/LightGBM/issues/740) as it may be 10x faster than MinGW especially for very large trees.
+- **Solution 8**: Please [use Visual Studio](https://github.com/Microsoft/LightGBM/issues/740) as it may be 10x faster than MinGW especially for very large trees.
```

Comment:
@Laurae2 Is existing reference to GitHub issue right? It leads to small python-package PR.